Dockerfile

```diff
@@ -37,6 +36,6 @@ ENV APP_ENV test
 # Ensure all python requirements are met
 RUN pip install -r /opt/app/requirements.txt
 
-CMD ["/bin/bash"]
-
 EXPOSE 8000
+
+CMD ["python","opt/app/bin/manage.py","runserver","0.0.0.0:8000"]
```

Comment:
Add the leading slash to the path so it is absolute. Add a space between elements in the array.